WatchApp Extension/Info.plist

```diff
@@ -26,13 +26,17 @@
 	<string>$(PRODUCT_MODULE_NAME).ComplicationController</string>
 	<key>CLKComplicationSupportedFamilies</key>
 	<array>
-		<string>CLKComplicationFamilyModularSmall</string>
 		<string>CLKComplicationFamilyCircularSmall</string>
 		<string>CLKComplicationFamilyExtraLarge</string>
-		<string>CLKComplicationFamilyUtilitarianSmallFlat</string>
-		<string>CLKComplicationFamilyUtilitarianLarge</string>
+		<string>CLKComplicationFamilyGraphicBezel</string>
```

Comment:
Looks like maybe you've also enabled the Graphic Rectangular and Graphic Bezel complications in settings?  Probably should leave those unchecked until there's code to support them.

Reply:
Thanks!  Good point.  I was working on complications for the Graphic Rectangular and Graphic Bezel, but haven't completed them yet; I should have unchecked them.